lib/server.js

```diff
@@ -395,12 +395,12 @@ class Server extends KarmaEventEmitter {
     }
 
     processWrapper.on('unhandledRejection', (error) => {
-      this.log.error('UnhandledRejection')
+      this.log.error(`UnhandledRejection: ${error.message}`)
```

Comment:
JS can throw any type, `Error` or not.  So I think these need to be eg `${error.message || String(error)}`

Reply:
If `error` is `undefined` or `null` for whatever reason, that line will have a new error trying to access `.message`. Hopefully that doesn't happen, but maybe it should be `(error && error.message) || String(error)`?